thanos-0.31.advisories.yaml

```diff
@@ -48,3 +48,10 @@ advisories:
         type: fixed
         data:
           fixed-version: 0.31.0-r6
+
+  - id: GHSA-m425-mq94-257g
```

Comment:
for this we are doing 

```
  - id: CVE-2023-44487
    aliases:
      - GHSA-m425-mq94-257g
```

Reply:
That is weird, wolfictl didn't gave me a CVE-ID

Reply:
that one is weird

Reply:
It appeared id: CVE-2023-44487 associated to another alias `GHSA-qppj-fm5r-hxr3` but when looking on the advisories data. You could find this relation 🤔.

Reply:
When looking at https://github.com/advisories/GHSA-m425-mq94-257g, it doesn't seem to have one associated to it.